server/repository/src/migrations/views/mod.rs

```diff
@@ -441,6 +442,47 @@ pub(crate) fn rebuild_views(connection: &StorageConnection) -> anyhow::Result<()
     -- Only show doses that haven't been deleted, unless they have a vaccination
     WHERE vcd.deleted_datetime IS NULL OR v.id IS NOT NULL;
 
+    CREATE VIEW vaccination_course AS 
+      SELECT
+        vc.id,
+        vc.name AS vaccine_course_name,
+        coverage_rate,
+        vc.is_active,
+        wastage_rate,
+        vc.deleted_datetime AS vaccine_course_deleted_datetime,
+        vcd.id AS vaccine_course_dose_id,
+        LABEL AS dose_label,
+        min_interval_days,
+        min_age,
+        max_age,
+        vcd.deleted_datetime AS vaccine_course_dose_deleted_datetime,
```

Comment:
Would it make sense to just filter out deleted in the view? Does this need to return deleted rows?